packages/graphql/src/classes/CallbackBucket.ts

```diff
@@ -62,12 +65,41 @@ export class CallbackBucket {
                         .split("\n")
                         .filter((line) => !line.includes(`$resolvedCallbacks.${cb.paramName}`))
                         .join("\n");
+                } else if (param === null) {
+                    params[cb.paramName] = null;
+                } else {
+                    params[cb.paramName] = this.parseCallbackResult(param, cb.type);
                 }
-
-                params[cb.paramName] = param;
             })
         );
 
         return { cypher, params };
     }
+
+    private parseCallbackResult(result: any, type: TypeMeta): any {
```

Comment:
We can use `CallbackReturnValue` from `packages/graphql/src/types/index.ts` as the return type here, as long as we add `Integer` to it. If that's not possible, then we can make a new type:

`type CallbackResult = Integer | number | string | boolean | null | CallbackResult[];`